docs/conf.py

```diff
@@ -274,6 +274,10 @@ def setup(app):
 
 html_js_files = [
     'js/custom.js',
+    (
+        'https://views.scientific-python.org/js/plausible.js',
+        {"data-domain": "pyhf.readthedocs.io", "defer": "defer"},
+    ),
```

Comment:
Following the advice of the [`sphinx-plausible`](https://github.com/rkdarst/sphinx-plausible) docs:

https://github.com/rkdarst/sphinx-plausible/blob/e869fe4a27533189170963fcf2d89acddc066d3b/README.md?plain=1#L14-L25